src/IO/InputReader.php

```diff
@@ -61,18 +61,10 @@ public function staticValue(
         $validator = $this->makeValidator($validator, $required);
         $answer = $this->io->askAndValidate($label, $validator, 3, $default);
 
-        if (!is_string($answer) && null !== $answer) {
-            return null;
-        }
-
         if (is_string($answer) && '' !== trim($answer)) {
             return trim($answer);
         }
 
-        if (is_string($default) && '' !== trim($default)) {
-            return trim($default);
-        }
-
```

Comment:
`Helper/QuestionHelper->getDefaultAnswer()` will trim the value already. So chance are slim that we're going to run into this.

```php
# QuestionHelper L170-174

$default = explode(',', $default);
foreach ($default as $k => $v) {
    $v = $question->isTrimmable() ? trim($v) : $v;
    $default[$k] = $choices[$v] ?? $v;
}